crypto/crypto.go

```diff
@@ -62,6 +62,11 @@ func CreateAddress(b common.Address, nonce uint64) common.Address {
 	return common.BytesToAddress(Keccak256(data)[12:])
 }
 
+// Creates an ethereum address given the bytes and the nonce
+func CreateAddress2(b common.Address, salt [32]byte, code []byte) common.Address {
+	return common.BytesToAddress(Keccak256([]byte{0xff}, b.Bytes(), salt[:], Keccak256(code))[12:])
```

Comment:
After changes to EIP-1014 this formulae changed a bit - now the code is not hashed. 
This also differs from SputnikVM implementation (see https://github.com/ETCDEVTeam/etcommon-rs/pull/67/files#diff-bce94f77b8183091af496690c60ca8e5R29).